packages/helpers/README.md

```diff
@@ -1 +1,7 @@
 # @plone/helpers
+
+## EXPERIMENTAL
```

Comment:
https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts

```suggestion
> [!WARNING]
> This package or app is experimental.
> The community offers no support whatsoever for it.
> Breaking changes may occur without notice.
```